frontend/src/components/AllQueues.js

```diff
@@ -36,23 +35,32 @@ const useStyles = makeStyles(theme => ({
   },
 }))
 
+//Forward ref used with MUI BaseButton. Does not allow observer.
 const QueueTabContent = React.forwardRef(({ onClick, queueData }, _ref) => {
+  const queueStore = useContext(QueueStoreContext)
   const classes = useStyles()
+
+  // Add state update functionality
+
+  // Add pinging for checking if queues have updated / websockets?
+
```

Comment:
can remove these lines

Reply:
The change in refreshAuthLogic is in reponse to prettier

Reply:
Took out comments